<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="grpManage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Управление списком репозиториев</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnLoadRepos">
         <property name="text">
          <string>Загрузить из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRepos">
         <property name="text">
          <string>Список добавленных репозиториев:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lstRepos"/>
       </item>
       <item>
        <widget class="QLabel" name="lblRepo">
         <property name="text">
          <string>Адрес репозитория:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineRepo"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddRepo">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabMain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actSaveList"/>
    <addaction name="actLoadTemplate"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actSaveList">
   <property name="text">
    <string>Сохранить список репозиториев</string>
   </property>
  </action>
  <action name="actLoadTemplate">
   <property name="text">
    <string>Загрузить макет job</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
